conf/solr/7.3.1/updateSchemaCMB.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+set -e
+
+# This script updates the <field> and <copyField> schema configuration necessary to properly
+# index custom metadata fields in Solr.
+# 1. Retrieve from Dataverse API endpoint
+# 2. Parse and write Solr schema files (which might replace the included files)
+# 3. Reload Solr
+#
+# List of variables:
+# ${DATAVERSE_URL}: URL to Dataverse. Defaults to http://localhost:8080
+# ${SOLR_URL}: URL to Solr. Defaults to http://localhost:8983
+# ${UNBLOCK_KEY}: File path to secret or unblock key as string. Only necessary on k8s or when you secured your installation.
+# ${TARGET}: Directory where to write the XML files. Defaults to /tmp
```

Comment:
Would it be possible to modify `scripts/api/setup-optional-harvard.sh` to call this script to show how to use it? That "setup-optional-harvard" script is used only for reference, to remind ourselves how we set up Harvard Dataverse.

Reply:
Done in scripts/api/setup-optional-harvard.sh